src/rad/wxfbmanager.cpp

```diff
@@ -155,16 +155,16 @@ wxObject* wxFBManager::GetWxObject( PObjectBase obj )
 
 void wxFBManager::ModifyProperty( wxObject* wxobject, wxString property, wxString value, bool allowUndo )
 {
-	CHECK_VISUAL_EDITOR()
+	CHECK_VISUAL_EDITOR(void())
```

Comment:
Why is this needed?

Reply:
This macro requires a parameter and MSVC spills out a warning if you don't supply one. There a tricks to produce macros that accept a variable amount of parameters or you can make two macros, one for methods that return something and one for methods that return void (wxWidgets uses this for its assert macros), but this was the least-amount-of-work solution.

Reply:
Can you disable the warning for MSVC instead?

Reply:
Interesting, according to the standard it is even an error to supply less arguments than required, but empty arguments are ok. For some reason [gcc](https://gcc.gnu.org/onlinedocs/cpp/Macro-Arguments.html) interprets empty parentheses as one empty argument but MSVC not, that one however doesn't produce an error if you don't supply enough.

Regarding that these macros are only used local in that file i guess i can disable the warning instead.